capabilities/writetarget/write_target_test.go

```diff
@@ -0,0 +1,95 @@
+package writetarget
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewWriteTargetID(t *testing.T) {
+	tests := []struct {
+		name            string
+		chainFamilyName string
+		networkName     string
+		chainID         string
+		version         string
+		expected        string
+		expectError     bool
+	}{
+		{
+			name:            "Valid input with network name",
+			chainFamilyName: "aptos",
+			networkName:     "mainnet",
+			chainID:         "1",
+			version:         "1.0.0",
+			expected:        "write_aptos-mainnet@1.0.0",
+			expectError:     false,
+		},
+		{
+			name:            "Valid input without network name",
+			chainFamilyName: "aptos",
+			networkName:     "",
+			chainID:         "1",
+			version:         "1.0.0",
+			expected:        "write_aptos-1@1.0.0",
+			expectError:     false,
+		},
+		{
+			name:            "Valid input with empty chainFamilyName",
+			chainFamilyName: "",
+			networkName:     "ethereum-mainnet",
+			chainID:         "1",
+			version:         "1.0.0",
+			expected:        "write-ethereum-mainnet@1.0.0",
```

Comment:
We need to keep it backward-compatible. Otherwise ,WF DON won't be able to find the WriteTarget
```suggestion
			expected:        "write_ethereum-mainnet@1.0.0",
```

Reply:
fixed in this commit: https://github.com/smartcontractkit/chainlink-evm/pull/2/commits/c0756fa4855f6fd70f3c4e8b141aa5e19bd1d344